usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -595,12 +606,19 @@ def select_columns_transaction_fabs_fpds(bronze_table_name):
             return select_cols
 
         def select_columns_transaction_normalized(bronze_table_name):
+            action_date_col = next(
+                filter(
+                    lambda c: c.dest_name == "action_date" and c.source == "action_date",
+                    FABS_TO_NORMALIZED_COLUMN_INFO if transaction_type == "fabs" else DAP_TO_NORMALIZED_COLUMN_INFO,
+                )
+            )
+            parse_action_date_sql_snippet = handle_column(action_date_col, bronze_table_name, is_result_aliased=False)
             select_cols = [
```

Comment:
Rather than calling `handle_column` here, why not just call `build_date_format_sql`?

Reply:
@DManowitz-BAH the rationale for using the `handle_column` to derive the date-parsing SQL for `action_date` rather than directly calling `build_date_format_sql` was two-fold:
1. to allow that when date-parsing `action_date`, or any other column that needs it, just pass in the `TransactionColumn` object and don't predetermine _what kind_ of date-parsing is needed. Let the logic of `handle_column` read the value of the `TransactionColumn.handling` to determine which of the (currently) two types of date-parsing to do. A bit of defensive programming: If not done this way, and casting to `DATE` is hard coded as `is_casted_to_date=True` or `is_casted_to_date=True` in a call to `build_date_format_sql`, if for some reason the `TranactionColumn.handling` value is changed in the column spec, it will probably lead to a subtle bug because it will be unlikely someone will remember to also go find any call to `build_date_format_sql` for that column and also change `True` to `False` or vice versa.
2. That way a general pattern is established for any other `TransactionColumn` object that needs to have date-parsing SQL built and extracted for use somewhere --> just pass the `TransactionColumn` object to `handle_column` and let it figure out based on the attributes of the object how to build the sql (e.g. cast or not cast)

Reply:
OK